tools/online_editor/todo/preview_image.cy.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright © SixtyFPS GmbH <info@slint-ui.com>
+// SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-commercial
+
+function test_demo(name: string) {
```

Comment:
was this file meant to be commited?  Why "todo" ?

Reply:
No, this one should not have ended up in here. I'll remove that again.